package.json

```diff
@@ -37,7 +37,7 @@
     "prepare": "install-peers",
     "test": "run-s test:circular lint format:check",
     "test:circular": "dpdm --warning false --tree false --exit-code circular:1 ./src/index.ts",
-    "lint": "eslint . --max-warnings=102 --color",
+    "lint": "eslint . --max-warnings=98 --color",
```

Comment:
This is by removing the error alert and changing it to  `AlertFormikError`.
I will see separately if there are more cases that we can remove like that.